### PR TITLE
add category list, opening by hover on it

### PR DESCRIPTION
**Pull request by kalipp1**
I had a lot of fun working on it. What you think about this idea?

### CODE REVIEW

**Comment by NocnyZwierz** [comment]
It would be good to add between categories border-bottom: solid. The width of the drop-down list must be equal to the button. It's good for the list to appear just below the button. :)

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 1]
```diff
@@ -3,15 +3,84 @@
...
+    ul {
+      padding: 0;
+      margin: 0;
+      list-style-type: none;
+      ul {
+        background-color: lightgray;
+        width: 100%;
+        position: absolute;
+        box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
+        li {
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:29]: please remove so deep nesting, use css classes instead

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 2]
```diff
@@ -3,15 +3,84 @@
...
+            display: block;
+            font-size: 14px;
+            border-bottom: 1px solid black;
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:32]: color as hex or rgb 

and from variable

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 3]
```diff
@@ -3,15 +3,84 @@
...
+      }
+    }
+
+    li:hover {
+      ul {
+          display: block;
+          position: absolute;
+          left: 50%;
+          transform: translateX(-50%);
+          background-color: lightgray;
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:59]: same, as above

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 4]
```diff
@@ -3,15 +3,84 @@
...
+      }
+      li {
+        display: block;
+        text-align: center;
+        background-color: lightgray;
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:64]: same, as above

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 5]
```diff
@@ -3,15 +3,84 @@
...
+        left: 50%;
+        transform: translateX(-50%);
+        padding: 10px;
+        a {
+            color: black;
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:69]: same, as above

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 6]
```diff
@@ -26,13 +95,26 @@
...
     select {
+      font-size: 14px;
       -webkit-appearance: none;
       border: 0;
       background-color: transparent;
       padding: 5px 30px 5px 35px;
       position: relative;
       z-index: 1;
+      outline: none;
+      option{
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:106]: add space before {

File: src/components/features/ProductSearch/ProductSearch.module.scss [hunk 7]
```diff
@@ -26,13 +95,26 @@
...
+      }
     }
+    select {
+      &:focus-visible {
+        outline: none;
+      }
+    }
+  }
+  .category:hover{
+    background-color: lightgray;
```
Comment (matwjablonski) [inline src/components/features/ProductSearch/ProductSearch.module.scss:117]: color

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Review by NocnyZwierz — APPROVED** [review]
Easy peasy lemon squeezy! Nice work!

**Review by NocnyZwierz — APPROVED** [review]

**Review by matwjablonski — APPROVED** [review]

**Review by NocnyZwierz — DISMISSED** [review]

**Review by matwjablonski — DISMISSED** [review]

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**